### PR TITLE
DS-2934 Add an empty "local" metadata namespace

### PR DESCRIPTION
**Pull request by helix84**
https://jira.duraspace.org/browse/DS-2934

### CODE REVIEW

File: dspace/config/registries/local-types.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <title>Registry of instance-specific metadata fields</title>
+    <contributor.author>Ivan Mas&#xE1;r</contributor.author>
+    <contributor.editor>$Author$</contributor.editor>
+    <date.created>$Date$</date.created>
+    <description>$Log$</description>
+    <description.version>$Revision$</description.version>
+  </dspace-header>
+
+  <dc-schema>
+    <name>dcterms</name>
```
Comment (KevinVdV) [inline dspace/config/registries/local-types.xml]: local-types.xml contains a reference to the dcterms schema.

**Comment by helix84** [comment]
Tested both ant fresh_install and ant update from 3.x -> 6.0, it works.

I found an oddity that when the DB is already the latest version, ant update won't load the registries. This is not caused by this PR and I encountered the same behavior in #1162.

**Comment by mwoodiupui** [comment]
+1

**Comment by terrywbrady** [comment]
+1

**Comment by tdonohue** [comment]
+1, needs documentation though. I'm not sure if the purpose behind this new namespace will be self-explanatory

**Pull request merged**